assets/src/edit-story/app/media/utils/useTranscodeVideo.js

```diff
@@ -28,8 +28,37 @@ import { useCurrentUser } from '../../currentUser';
 
 const isDevelopment = process.env.NODE_ENV === 'development';
 
+/**
+ * Returns file basename without extension.
+ *
+ * @param {File} file File object.
+ * @param {string} file.name File name.
+ * @return {string} File name without extension.
+ */
 const getFileName = ({ name }) => name.split('.').slice(0, -1).join('.');
 
+/**
+ * Determines whether it's a video file.
+ *
+ * @param {File} file File object.
+ * @param {string} file.type File type.
+ * @return {boolean} Whether it's a video file.
+ */
+const isVideo = ({ type }) => type.startsWith('video/');
+
+const GB_IN_BYTES = 1024 * 1024 * 1024;
```

Comment:
I would move this to the consts file.

Reply:
It's only used in one place, so there's not really a need to extract it. Once it's needed in more places it can be done easily.

Reply:
I was thinking of using it here - https://github.com/google/web-stories-wp/pull/6189#discussion_r570347934